src/utils/api.ts

```diff
@@ -113,9 +113,28 @@ export const offlineDownload = (
   return r.post(`/fs/add_${type}`, { path, urls })
 }
 
+export const helper = async (url: string) =>{
```

Comment:
you can move this function to `/utils/str.ts`

Reply:
fixed